### ISSUE TITLE
Support optional .env file for development / production config

### ISSUE THREAD

**Issue by jimpick** [ipfs-shipyard/peer-pad#256]
I'd like to be able to tweak the app name and the rendezvous server multiaddr in development.

Those are also settings we'd probably want to change in various test / staging situations.

What do people think about using a .env file? I've used it in many projects.

https://github.com/motdotla/dotenv

**Comment by victorb** [comment]
Yes! `.env` are very pleasant and would be great to support. Same for peer-star-app itself.

**Comment by parkan** [comment]
+1 for dotenv, it's one standard most people can agree on :)
